clap_derive/src/derives/args.rs

```diff
@@ -361,7 +361,7 @@ pub fn gen_augment(
 
 fn gen_arg_enum_possible_values(ty: &Type) -> TokenStream {
     quote_spanned! { ty.span()=>
-        .possible_values(&<#ty as clap::ArgEnum>::VARIANTS)
+        .possible_values(<#ty as clap::ArgEnum>::VARIANTS)
```

Comment:
Not sure about that change, but it was needed for the macro tests to compile. We had `&&` otherwise and that did not fit well with IntoIterator

Reply:
I think it makes sense